src/components/Dialog-Window/Resources/DialogResources.module.scss

```
@@ -0,0 +1,23 @@
+.li,
+a {
+  color: rgba(31, 103, 199, 1);
+}
+
+.inputContainer {
+  display: flex;
+}
+
+.input {
+  flex-grow: 0;
```

Flex-grow: 1 will make the element take up the remaining space

```suggestion
  flex-grow: 1;
```

---

With `display: flex  `the input element will take up the entire width of the screen on PC as well. That looks a bit strange?

---

Yeah, we should add a `max-width: 19.5rem` to the container as well

---

Or just keep it as is

---

We can add a max-width to the container